package.json

```diff
@@ -3,9 +3,10 @@
   "homepage": "http://postcss.com/",
   "scripts": {
     "start": "babel-node scripts/build --server --dev --open",
-    "static": "babel-node scripts/build --static --production",
+    "static": "npm run lint && babel-node scripts/build --static --production",
```

Comment:
you shoud instead use `"pretest": "npm run lint"` to lint before test (yeah test will call "static" anyway) ^^